guides/source/configuring.md

```diff
@@ -465,7 +465,7 @@ encrypted cookies salt value. Defaults to `'signed encrypted cookie'`.
   method should be performed on the parameters. See [Security Guide](security.html#unsafe-query-generation)
   for more information. It defaults to `true`.
 
-* `config.action_dispatch.rescue_responses` configures what exceptions are assigned to an HTTP status. It accepts a hash and you can specify pairs of exception/status. By default, this is defined as:
+* `config.action_dispatch.rescue_responses` configures what exceptions are assigned to an HTTP status. It accepts a hash and you can specify pairs of exception/status. Please note that this is only for the purposes of logging. No exceptions are actually rescued. By default, this is defined as:
```

Comment:
That's rather a matter of perspective, I think...

The exception will still lead to an error response, but it is rescued from being a generic 500, and instead processed by a handler that will return a more pertinent HTTP status.